velox/expression/tests/CastExprTest.cpp

```diff
@@ -569,6 +570,20 @@ TEST_F(CastExprTest, arrayCast) {
   }
 }
 
+TEST_F(CastExprTest, arrayCastTest) {
```

Comment:
Looks like this added test case is not relevant to DECIMAL casting. Shall we make it a separate PR if it's necessary?

Reply:
Looks like I accidentally moved things around. This is not relevant to this PR. Will remove this.